src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped its work");
```

Comment:
Similar to the 'doWork' method, the message in 'stopWork' should be more informative and indicate the specific type of machine and the state of its work.

Reply:
Fixed